strawberryfields/gbs/points.py

```diff
@@ -101,8 +121,8 @@ def sample(K: np.ndarray, n_mean: float, n_samples: int) -> list:
      [0, 0, 0, 0]]
 
     Args:
-        K (array): the kernel matrix
-        n_mean (float): average number of points
+        K (array): the positive semidefinite kernel matrix
+        n_mean (float): average number of points per sample
```

Comment:
Just highlighting that I made these changes so would be good for someone to take a look